content/attacks/posts/Create - 2023-11-23-HecoCrossChainBridge&HTX.md

```diff
@@ -0,0 +1,37 @@
+---
+date: 2023-11-23
+target-entities: Heco Cross Chain Bridge and HTX
+entity-types:  
+   - Exchange
+   - Bridge
+attack-types:
+   - Private Key Leak
+title: “HECO Chain bridge compromised over $86.6M sent to suspicious addresses”
+loss: 87000000
+---
+
+## Summary
+
+Blockchain [security firm’s](https://blockworks.co/news/htx-hack-ethereum-crypto-assets) CertiK, Peckshield, and Cyvers have reported over $86.6 million in digital assets, including stablecoins, ETH, SHIB, and LINK, have been transferred from the HECO Chain bridge to suspicious addresses. Various other ERC-20 tokens were also drained, swapped for ether, and distributed to at least eight wallets. Withdrawals and deposits have been temporarily [suspended](https://cointelegraph.com/news/heco-chain-bridge-hack-86-million-lost) as the company investigates the incident, with services expected to resume after the investigation is completed. 
+
+## Attackers
+
+The attacker has yet to be identified. 
```

Comment:
There are plenty of addresses the funds were transferred initially, add couple of them